docs/src/userguide/numpy_tutorial.rst

```diff
@@ -472,21 +367,16 @@ here has gone through a lot of thinking and planning already; still they may
 or may not happen depending on available developer time and resources for
 Cython.
 
-1. Support for efficient access to structs/records stored in arrays; currently
-   only primitive types are allowed.
-2. Support for efficient access to complex floating point types in arrays. The
+1. Support for efficient access to complex floating point types in arrays. The
    main obstacle here is getting support for efficient complex datatypes in
    Cython.
-3. Calling NumPy/SciPy functions currently has a Python call overhead; it
+2. Calling NumPy/SciPy functions currently has a Python call overhead; it
    would be possible to take a short-cut from Cython directly to C. (This does
    however require some isolated and incremental changes to those libraries;
    mail the Cython mailing list for details).
```

Comment:
Cython maps some of the operations to C calls in `numpy/__init__.pxd` (a.k.a. `cimport numpy`). That's not what this comment refers to, but it's not like there's nothing there at this front...
I'd remove that comment, also because it's fairly unclear and unhelpful overall.